adal/src/main/java/com/microsoft/aad/adal/AuthorityValidationMetadataCache.java

```diff
@@ -57,8 +57,8 @@
 
     private static ConcurrentMap<String, InstanceDiscoveryMetadata> sAadAuthorityHostMetadata = new ConcurrentHashMap<>();
 
-    private AuthorityValidationMetadataCache() {
-        // DateExtensions class, no public constructor
+    protected AuthorityValidationMetadataCache() {
```

Comment:
Why does a utility class need a `protected` constructor?

Reply:
changed back to private